test/Microsoft.OpenApi.Tests/Services/OpenApiInfoComparerTests.cs

```diff
@@ -0,0 +1,297 @@
+                                    Id = "schemaObject1"
+                                }
+                            }
+                        }
+                    }
+                },
+                RequestBodies = new Dictionary<string, OpenApiRequestBody>
+                {
+                    ["requestBody1"] = new OpenApiRequestBody
+                    {
+                        Description = "description",
+                        Required = true,
+                        Content =
+                        {
+                            ["application/json"] = new OpenApiMediaType
+                            {
+                                Schema = new OpenApiSchema
+                                {
+                                    Reference = new OpenApiReference
+                                    {
+                                        Id = "schemaObject1",
+                                        Type = ReferenceType.Schema
+                                    }
+                                }
+                            }
+                        }
+                    },
+                    ["requestBody2"] = new OpenApiRequestBody
+                    {
+                        Description = "description",
+                        Required = true,
+                        Content =
+                        {
+                            ["application/json"] = new OpenApiMediaType
+                            {
+                                Schema = new OpenApiSchema
+                                {
+                                    Reference = new OpenApiReference
+                                    {
+                                        Id = "schemaObject1",
+                                        Type = ReferenceType.Schema
+                                    }
+                                }
+                            }
+                        }
+                    }
+                }
+            }
+        };
+
+        public OpenApiInfoComparerTests(ITestOutputHelper output)
+        {
+            _output = output;
+        }
+
+        public static IEnumerable<object[]> GetTestCasesForOpenApiInfoComparerShouldSucceed()
+        {
+            // Differences in ContentType,Style,Explode and AllowReserved
+            yield return new object[]
+            {
+                "Differences in ContentType,Style,Explode and AllowReserved",
+                new OpenApiEncoding
+                {
+                    ContentType = "image/png, image/jpeg",
+                    Style = ParameterStyle.Simple,
+                    Explode = true,
+                    AllowReserved = true
+                },
+                new OpenApiEncoding
+                {
+                    ContentType = "image/jpeg",
+                    Style = ParameterStyle.Form,
+                    Explode = false,
+                    AllowReserved = false
+                },
```

Comment:
This doesn't look right. Copy-paste error?

Reply:
sorry what doesn't seem right?

---
In reply to: [223510742](https://github.com/Microsoft/OpenAPI.NET/pull/328#discussion_r223510742) [](ancestors = 223510742)

Reply:
oh got it yes copy paste error for sure

---
In reply to: [223851085](https://github.com/Microsoft/OpenAPI.NET/pull/328#discussion_r223851085) [](ancestors = 223851085,223510742)